app/src/main/java/com/maurosarti/mvpcalculator/util/bus/observers/CalculatorButtonPressedBusObserver.java

```diff
@@ -0,0 +1,21 @@
+package com.maurosarti.mvpcalculator.util.bus.observers;
+
+public abstract class CalculatorButtonPressedBusObserver extends BusObserver<CalculatorButtonPressedBusObserver.CalculatorButtonPressed> {
+
+    public CalculatorButtonPressedBusObserver() {
+        super(CalculatorButtonPressed.class);
+    }
+
+    public static class CalculatorButtonPressed {
+
+        private String digit;
+
+        public CalculatorButtonPressed(String digit) {
+            this.digit = digit;
+        }
+
+        public String getDigit(){
+            return this.digit;
```

Comment:
only as best practice, not neccesary to change.

Not use keyword this when you not need